src/test/java/com/sampleweb/journey/DragosTests.java

```diff
@@ -0,0 +1,56 @@
+package com.sampleweb.journey;
+
+import com.sampleweb.BaseTest;
+import com.sampleweb.framework.PageNavigator;
+import com.sampleweb.pages.BooksPage;
+import org.openqa.selenium.By;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.awt.print.Book;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+
+public class DragosTests extends BaseTest {
+
+    private PageNavigator navigator = new PageNavigator();
+    private BooksPage BooksPage;
+
+    @BeforeMethod
+    public void setUp() throws Exception {
+        BooksPage = navigator.navigateToPage(driver, com.sampleweb.pages.BooksPage.PATH, BooksPage.class);
```

Comment:
why are you using the fully qualified name when you are importing the class anyway?